### DISCUSSION TITLE
/Quatt?uor/

### DISCUSSION THREAD

**Discussion by ajmalton** [DivinumOfficium/divinum-officium#2563] [General]
During a pass through the code, I introduced, then eliminated, a fault occasioned by the spelling of _quatuor_.

The classical spelling has -tt- but the later language writes -t- . As in my nova & vetera. In the database we have 257 quatuor and 360 quattuor.  Although the spelling could be regularized, it seems to me that it will diverge again.

So, to be on the safe side, let's standardize on /quatt?uor/i in the code, whenever this is tested.  There are currently 13 places, and they all only test Quattuor.

**Comment by APMarcello3** [comment]
The 1960 text has "Quatuor", whereas previous versions have "Quattuor". It would be ideal if we could have a solution much like already exists in the code, where the letter J is replaced with I for 1960 rubrical structures.

**Reply by ajmalton** [reply]
Displaying Quatuor everywhere when using the 1960 rubric, is doable.

Both forms occur in the proper texts and also crucially in the names of days, in our data. Thus testing for it (eg to condition on ember days) should always test for both.
